src/lowlev2.c

```diff
@@ -93,7 +93,7 @@ LispPTR N_OP_getbasebyte(register LispPTR base_addr, register int byteoffset) {
 
 ***********************************************************/
 
-LispPTR N_OP_putbasebyte(register LispPTR base_addr, register int byteoffset, register int tos) {
+LispPTR N_OP_putbasebyte(LispPTR base_addr, LispPTR byteoffset, LispPTR tos) {
```

Comment:
not sure I understand why they were int before and LispPTR now

Reply:
Probably because back in 2017 (or earlier) when I looked at the code I didn't connect up the fact that the other arguments being passed were NOT just numbers but were actually "LispPTR" types, pulled from the stack, from which an integer values was going to be extracted.  In the real original code all Lisp objects were just treated as "int", all the machines were 32-bit, and arithmetic and byte/bit operations were just assumed to do something that would work without regard to signedness... and the compilers didn't diagnose any of that as a potential problem.